code/game/dna/genes/vg_powers.dm

```diff
@@ -237,6 +237,16 @@
 	if(!say)
 		return
 	say = strip_html(say)
+	say = replacetext(say, "\[b\]",		"<B>")
```

Comment:
Why not actually use the papercode procs? https://github.com/ParadiseSS13/Paradise/blob/master/code/__HELPERS/text.dm#L459-L516